### ISSUE TITLE
Add "Installation" instructions to readme

### ISSUE THREAD

**Issue by WorldsEndless** [gsomoza/firefox-easy-container-shortcuts#25]
You have "Usage", but how to install into my Firefox, other than the official Firefox listing?

**Comment by gsomoza** [comment]
Good idea to add installation instructions in general, but I would just link to the Firefox docs for the specific need you mentioned. To install from outside the Firefox listing you could [try this](https://extensionworkshop.com/documentation/publish/install-self-distributed/#install-addon-from-file-android).

**Comment by gsomoza** [comment]
Reopening cause we could add a generic installation instructions section with links to the official docs.
